dev/db2/oldest/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  db2-oldest:
+    container_name: sequelize-db2-oldest
+    image: ibmcom/db2-amd64:11.5.6.0a
+    privileged: true
+    env_file: ../.env_list
+    environment:
+      DBNAME: testdb
```

Comment:
There's a comment in the old start.sh that db names should be uppercase. Is that comment invalid or do we make this uppercase somewhere else in the code?

Reply:
I'm unsure on this. The only reason I left it lower case is because this is what the [db2 docs](https://www.ibm.com/docs/en/db2/11.5?topic=system-linux) (see step 6) used.

Reply:
In that case I think it should be fine